packages/features/ee/round-robin/roundRobinManualReassignment.ts

```diff
@@ -134,8 +138,7 @@ export const roundRobinManualReassignment = async ({
     const responseSafeParse = await responseSchema.safeParseAsync(bookingResponses);
     const responses = responseSafeParse.success ? responseSafeParse.data : undefined;
 
-    let bookingLocation = booking.location;
-    if (eventType.locations.includes({ type: OrganizerDefaultConferencingAppType })) {
```

Comment:
includes compare object by reference and not value so this won't ever be true even when it contains an object `{ type: OrganizerDefaultConferencingAppType }`